homework01/test/examples/Main.hs

```diff
@@ -1,15 +1,44 @@
-module Main
-  ( main
-  )
-  where
-
--- doctest
-import qualified Test.DocTest as DocTest
+module Main where
 
+import Basis
+import Test.Hspec
 
 main :: IO ()
-main =
-  DocTest.doctest
-    [ "-isrc"
-    , "src/Basis.hs"
-    ]
+main = hspec $ do
+
+  describe "toDigits" $ do
+    it "converts positive integers to a list of digits" $
+      toDigits 1234 `shouldBe` [1, 2, 3, 4]
+
+    it "returns an empty list for 0" $
+      toDigits 0 `shouldBe` []
+
+    it "returns an empty list for negative inputs" $
+      toDigits (-17) `shouldBe` []
+
+  describe "toDigitsRev" $
```

Comment:
What does this function makes if a negative number is passed as argument? and like before, not sure if important but what if the number starts with `0`